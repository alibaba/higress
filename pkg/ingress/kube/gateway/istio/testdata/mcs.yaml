apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: gateway
  namespace: higress-system
spec:
  addresses:
  - value: higress-gateway
    type: Hostname
  gatewayClassName: higress
  listeners:
  - name: default
    port: 34000
    protocol: TCP
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: TCPRoute
metadata:
  name: tcp
  namespace: higress-system
spec:
  parentRefs:
  - name: gateway
    namespace: higress-system
  rules:
  - backendRefs:
    - group: multicluster.x-k8s.io
      kind: ServiceImport
      name: echo
      port: 80
