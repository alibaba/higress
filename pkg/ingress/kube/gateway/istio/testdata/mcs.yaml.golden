apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: higress-gateway.higress-system.svc.domain.suffix
    internal.istio.io/parents: Gateway/gateway/default.higress-system
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway-default
  namespace: higress-system
spec:
  servers:
  - hosts:
    - higress-system/*
    port:
      name: default
      number: 34000
      protocol: TCP
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parents: TCPRoute/tcp.higress-system
    internal.istio.io/route-semantics: gateway
  creationTimestamp: null
  name: tcp-tcp-istio-autogenerated-k8s-gateway
  namespace: higress-system
spec:
  gateways:
  - higress-system/gateway-istio-autogenerated-k8s-gateway-default
  hosts:
  - '*'
  tcp:
  - route:
    - destination:
        host: echo.higress-system.svc.domain.suffix
        port:
          number: 80
---
